libcxx/utils/ci/Dockerfile

```diff
@@ -22,12 +22,11 @@
 # If you're only looking to run the Docker image locally for debugging a
 # build bot, see the `run-buildbot-container` script located in this directory.
 #
-# A pre-built version of this image is maintained on DockerHub as ldionne/libcxx-builder.
-# To update the image, rebuild it and push it to ldionne/libcxx-builder (which
-# will obviously only work if you have permission to do so).
+# A pre-built version of this image is maintained on Github under the libc++ organization, as ghcr.io/libcxx/libcxx-builder.
+# To update the image, rebuild it and push it to github (all members of the libc++ organization should be able to do this).
 #
-#   $ docker build -t ldionne/libcxx-builder libcxx/utils/ci
-#   $ docker push ldionne/libcxx-builder
+#   $ docker compose build
```

Comment:
There is "docker-compose" and "docker compose"
On Ubuntu 22.04 (the lastest LTS version)  "docker compose" is not available.
This link contains more information.
https://github.com/docker/compose-cli/issues/1404#issue-823883873

Can you add a comment that "docker-compose" can be used when "docker compose" is not available?

Reply:
I don't know that it can be, I can't test. And I think we want to require modern enough versions of docker to use modern features.

You can get docker compose on Ubuntu using the official install scripts. [1] It's how I get docker on debian. And while using additional package repos is normally smelly, it seems to be the norm for docker.

```
curl -fsSL https://get.docker.com -o get-docker.sh
sudo sh get-docker.sh
```

[1] https://docs.docker.com/engine/install/ubuntu/